### PR TITLE
Fix #2681 - orphaned variable

### LINKED ISSUE #2681: orphaned variable
In v1.8.10 member.php line 1099, `$emailnotifycheck = '';` is unused, and a useless variable. It can be safely removed.

Original thread: [orphaned variable](https://community.mybb.com/thread-208421.html)

### PR DESCRIPTION
**Pull request by SvePu**
#2681 orphaned variable

### CODE REVIEW

File: member.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
 $templatelist .= ",member_register_question,member_register_question_refresh,usercp_options_timezone,usercp_options_timezone_option,usercp_options_language_option,member_register_language,member_profile_customfields_field_multi_item,member_profile_customfields_field_multi";
 $templatelist .= ",member_profile_contact_fields_aim,member_profile_contact_fields_google,member_profile_contact_fields_icq,member_profile_contact_fields_skype,member_profile_contact_fields_yahoo,member_profile_pm,member_profile_contact_details,member_profile_banned";
-$templatelist .= ",member_profile_banned_remaining,member_profile_addremove,member_emailuser_guest,member_register_day,usercp_options_tppselect_option,postbit_warninglevel_formatted,member_profile_userstar,member_profile_findposts,member_profile_findthreads";
+$templatelist .= ",member_profile_banned_remaining,member_profile_addremove,member_emailuser_guest,member_register_day,usercp_options_tppselect_option,postbit_warninglevel_formatted,member_profile_userstar,member_profile_findposts,member_profile_findthreads";
```
Comment (euantorano) [inline member.php]: I can't see anything different here, am I being blind?

Comment (Stefan-ST) [inline member.php]: I don't see a difference either...

Comment (skmedix) [inline member.php]: Even my diff checker said these two lines are identical.

Comment (SvePu) [inline member.php]: I did no changes into this variable.
Don't know why there's a diff report.

Comment (Cu8eeeR) [inline member.php]: It is probably bug in GitHub... can you try to make a new PR? No big deal, just know what causes that

Comment (Matslom) [inline member.php]: It's not GitHub bug, once I had same issue in git kraken.

Comment (euantorano) [inline member.php]: We'll just live with it, it's not a major issue.

**Pull request merged**